src/main/java/org/opentripplanner/routing/edgetype/PathwayEdge.java

```diff
@@ -23,37 +22,56 @@
     public static final I18NString DEFAULT_NAME = new NonLocalizedString("pathway");
 
     private final I18NString name;
-    private int traversalTime;
-    private double distance;
-    private int steps;
-    private double angle;
-
-    private boolean wheelchairAccessible = true;
-    private FeedScopedId id;
-
-    public PathwayEdge(Vertex fromv, Vertex tov, I18NString name) {
-        super(fromv, tov);
-        this.name = Objects.requireNonNullElse(name, DEFAULT_NAME);
+    private final int traversalTime;
+    private final double distance;
+    private final int steps;
+    private final double slope;
+
+    private final boolean wheelchairAccessible;
+    private final FeedScopedId id;
+
+    public static PathwayEdge lowCost(Vertex fromV, Vertex toV, I18NString name) {
```

Comment:
Could you add some documentation here, what `lowCost` means?

Reply:
Added.